Marlin/Marlin_main.cpp

```diff
@@ -10185,6 +10203,104 @@ inline void gcode_M502() {
 
 #endif // DUAL_NOZZLE_DUPLICATION_MODE
 
+#if ENABLED(FILAMENT_LOAD_UNLOAD_GCODES)
+
+  /**
+   * M701: Load filament
+   *
+   *  T[extruder] - Optional extruder number. Current extruder if omitted.
+   *  L[distance] - Extrude distance for insertion (positive value) (manual reload)
+   *
+   *  Default values are used for omitted arguments.
+   *
+   */
+  inline void gcode_M701() {
+
+    if (get_target_extruder_from_command(200)) return;
+
+    // Load filament
+    const float load_length = parser.seen('L') ? parser.value_axis_units(E_AXIS) : 0
+      #ifdef FILAMENT_CHANGE_LOAD_LENGTH
+        + FILAMENT_CHANGE_LOAD_LENGTH
+      #endif
+    ;
+
+    #if ENABLED(MK2_MULTIPLEXER)
+      if (target_extruder != active_extruder) select_multiplexed_stepper(target_extruder);
+    #endif
+
+    load_filament(target_extruder, load_length, ADVANCED_PAUSE_EXTRUDE_LENGTH, FILAMENT_CHANGE_NUMBER_OF_ALERT_BEEPS,
+                  true, false, ADVANCED_PAUSE_MODE_LOAD_FILAMENT);
+
+    #if ENABLED(MK2_MULTIPLEXER)
+      if (target_extruder != active_extruder) select_multiplexed_stepper(active_extruder);
+    #endif
+  }
+
+  /**
+   * M702: Unload filament
+   *
+   *  T[extruder] - Optional extruder number. Current extruder (or all extruders if FILAMENT_UNLOAD_ALL_EXTRUDERS
+                    is enabled) if omitted.
+   *  Z[distance] - Move the Z axis by this distance
+   *  U[distance] - Retract distance for removal (positive value) (manual reload)
+   *
+   *  Default values are used for omitted arguments.
+   *
+   */
+  inline void gcode_M702() {
+    point_t park_point = NOZZLE_PARK_POINT;
+
+    if (get_target_extruder_from_command(702)) return;
+
+    // Z axis lift
+    if (parser.seenval('Z')) park_point.z = parser.linearval('Z');
+
+    // Unload filament
+    const float unload_length = parser.seen('U') ? -parser.value_axis_units(E_AXIS) : 0
```

Comment:
Is this for consistency with bq fork of Marlin? I suggest we save all confusion and do…
```cpp
const float unload_length = -FABS(parser.seen('U') ? parser.value_axis_units(E_AXIS) : 0
  #if FILAMENT_CHANGE_UNLOAD_LENGTH
    + (FILAMENT_CHANGE_UNLOAD_LENGTH)
  #endif
);
```